app/api/v2/admin/permissions.rb

```diff
@@ -102,15 +102,15 @@ def validate_params!(params)
           ]
           params do
             requires :id,
-                     type: String,
+                     type: Integer,
                      allow_blank: false,
                      desc: 'Permission id'
             optional :role,
                      type: String,
                      allow_blank: false,
                      desc: 'permission field - role'
             optional :req_type,
```

Comment:
should be named `:verb`
There is no such field `req_type` in DB
And add a test for that, please